README.md

```diff
@@ -15,18 +15,32 @@ The KBase Narrative Interface builds on the [Jupyter Notebook](http://jupyter.or
 
 This document contains links to various documentation in the [docs](docs) directory, with a brief description of each.
 
+## Installation
+If you want to use the KBase Narrative Interface, just point your browser at https://narrative.kbase.us, make a free account, and jump in. This repo is only for people who wish to contribute to the development of the interface.
+
 ## Local Installation (for developers)
 Short version:
 Requires the following:
-* Python 2.7+ (working on updating to Python 3...)
-* Node.js v6 LTS (needed for npm)
-* Bower
+* Python 3.6+
+* Anaconda/Miniconda as an environment manager (https://www.anaconda.com/)
+* Node.js (latest LTS recommended)
+* Bower 1.8.8+
 
+### Using a conda environment (recommended!)
 ```
 git clone https://github.com/kbase/narrative
 cd narrative
-./scripts/install_narrative.sh -v narrative_venv
-source narrative_venv/bin/activate
+conda env -n my_narrative_environment
```

Comment:
Looks like you're missing a verb here...

Reply:
That's pretty well jumbled anyway. I'm gonna close this PR and try some fixes on the develop branch.